src/utils/notificationUtils.js

```diff
@@ -7,23 +9,37 @@ export const getNotificationProps = notification => {
   const user2Name =
     notification?.message_values?.user2_name || 'Unnamed User';
 
-  // TODO flesh out more once this is included in notifications DEX-739
-  const individual1AdoptionName = deriveIndividualName(
-    notification,
-    'AdoptionName',
-    'Unnamed individual',
+  const yourIndividualName = 'one of the individuals';
+  // notification?.message_values?.user2_name || 'Unnamed Individual'; //TODO update after DEX-927 is resolved
+  const theirIndName =
+    notification?.message_values?.target_individual_name ||
+    'Unnamed Individual';
+  const theirIndividualGuid =
+    notification?.message_values?.target_individual_guid;
+  const theirIndividualName = (
+    <span>
+      <Link
```

Comment:
I'm not sure that JSX belongs in utils files.

Reply:
I don't see an easy alternative with this one, where I needed to Linkify the value passed to something in en.json.

Reply:
yeah, this is not a util, it's a component.

Reply:
Fixed. Done.